wzdx/tools/polygon_tools.py

```diff
@@ -3,6 +3,7 @@
 import numpy as np
 import pyproj
 import logging
+import math
```

Comment:
Several of these imports are unused, can we clean up?

Reply:
I just went through and removed all unused imports, and sorted all import sections